Formula/bats-core.rb

```diff
@@ -0,0 +1,28 @@
+class BatsCore < Formula
+  desc "Bash Automated Testing System"
+  homepage "https://github.com/bats-core/bats-core"
+  url "https://github.com/bats-core/bats-core/archive/v1.0.1.tar.gz"
+  sha256 "821626f1e5058a4f25a95722399b460942f27535186a815a279e518b503f8de7"
+
+  bottle :unneeded
+
+  conflicts_with "bats@0.4",
+    :because => "bats@0.4 and bats-core install a `bats` executable"
```

Comment:
Nit: "both install a `bats` executable" is slightly shorter, and is what we use elsewhere for this type of conflict :slightly_smiling_face:

Reply:
Fix, there is no bats formula to conflic with any more, so I removed the block :)